src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/TaskCache.cs

```diff
@@ -40,5 +40,14 @@ private static Task<int>[] CreateInt32Tasks()
 
             return tasks;
         }
+
+        /// <summary>
+        /// A cached task for default(TResult).
+        /// </summary>
+        internal static class CachedDefaultValueTask<TResult>
```

Comment:
Nit: CachedDefaultValueTask => CachedDefaultTask... otherwise it can be read as "cached default ValueTask" rather than "cached "default value" task"